src/modules/home/index.tsx

```diff
@@ -1,16 +1,29 @@
 import { lazyRoute } from '#/utils/router.util';
 import { type RouteObject } from 'react-router-dom';
-import { WishlistRoute } from './routes/wishlist.route';
+import { AuthLayout } from '#/routes/layout-protected.route';
 
 const { HomeRoute } = lazyRoute(() => import('./routes/home.route'), 'HomeRoute');
+const { WishlistRoute } = lazyRoute(() => import('./routes/wishlist.route'), 'WishlistRoute');
 
 export const HomeRoutes: RouteObject[] = [
   {
     path: '/',
-    element: <HomeRoute />,
+    element: <AuthLayout protectedRoute={false} />,
+    children: [
+      {
+        path: '',
+        element: <HomeRoute />,
+      },
+    ],
   },
```

Comment:
Create a function that generates this object, and use it for each route.

Reply:
commented below on protected routes